caliban/config/__init__.py

```diff
@@ -137,6 +137,8 @@ def expand_image(image: str) -> str:
     'password': str,
     'artifact_root': str,
     s.Optional('debug'): bool,
+    s.Optional('pubsub_project'): str,  # default = project above
+    s.Optional('pubsub_topic', default='mlflow'): str,
```

Comment:
do we want to namespace this, like `caliban.uv.mlflow`?

Reply:
done